jpype/_core.py

```diff
@@ -140,14 +181,14 @@ def startJVM(*args, **kwargs):
     The default jvmpath is determined by ``jpype.getDefaultJVMPath()``.
 
     Parameters:
-     *args (Optional, str[]): Arguments to give to the JVM.
-        The first argument may be the path the JVM.
+     *jvmargs (Optional, str[]): Arguments to give to the JVM.
```

Comment:
this is an API breaking change, which I'd like to avoid (without a prior deprecation).

Reply:
I don't see the breaking change part, could you elaborate?

Perhaps you missed the fact that it was proceeded with ``*``, and therefore not a named argument in any way? There is no way that the name of the variable arguments component leaks out into the signature - I have changed it here entirely for readability reasons (since it really is just ``jvmargs`` that you can pass through).

Reply:
absolutely, eventually I should just rest with my broken bone and leave this stuff to @Thrameos. Sorry.

Reply:
I think there may be a problem with this after-all. It was previously allowed to call ``startJVM([None])``

There is a PR incoming.